packages/retail-ui/components/Loader/Loader.tsx

```diff
@@ -24,6 +24,7 @@ export interface LoaderProps {
   caption?: SpinnerProps['caption'];
   className?: string;
   type?: 'mini' | 'normal' | 'big';
+  component?: React.ReactNode | (() => React.ReactNode);
```

Comment:
Этого должно быть достаточно
```suggestion
  component?: React.ReactNode;
```